index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
```

Comment:
You are missing the MS `ie=edge` meta tag. This is still necessary until IE11 rules the world.